content/events-api/overview.md

```diff
@@ -1,4 +1,11 @@
-# Overview
+# 👋 Welcome in the Events API basics documentation!
+
+You are probably asking yourself a ton of questions about our Events API, that's why we made this article.
+
+::: warning
+The Events API feature is only available in the **5.0 version** and higher and for **SaaS** customers.
```

Comment:
```suggestion
The Events API feature is only available since the **5.0 version** and for **SaaS** customers.
Please note that the events API feature is not currently available for **Apps**.
```